src/common/perf_counter_connection.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "common/perf_counters.h"
+#include "common/Mutex.h"
+
+class CephContext;
+/* PerfcounterCollection hold the lock for PerfCounterCollectionImpl */
```

Comment:
no need to repeat the comment here. we could just put amend the comment right before `m_lock`.